tox.ini

```diff
@@ -22,12 +22,12 @@ commands =
 [testenv:py27-functional]
 commands =
    python -V
-   {toxinidir}/scripts/kube-init.sh nosetests -v []
```

Comment:
Switched them all except for coverage. Let's wait on that one to see if it runs into trouble